src/main/java/pb360/service/impl/GenerateRestApiServiceImpl.java

```diff
@@ -105,9 +107,13 @@ private void readJson(String link) {
 			jsonNodeList.add(readService(jsonObject));
 			jsonNodeList.add(readJUnitTest(jsonObject));
 			jsonNodeList.add(readController(jsonObject));
+			jsonNodeList.add(readCommon(jsonObject));
 
 			jsonDataModel.setTitle(jsonObject.get("title").toString());
 			jsonDataModel.setNodes(jsonNodeList);
+			
+			
+			writeModelfile(jsonDataModel);
```

Comment:
please change this method to common method, we need loop data on JsonDataModel and write to output, please find the way reuse code when make any function